README.md

```diff
@@ -1,28 +1,153 @@
+   id                SERIAL PRIMARY KEY,
+   country_code      varchar(2) not null UNIQUE,
+   multiplier        double precision NOT NULL,
+   updated_at        timestamp NOT NULL default now()
+);
+CREATE TABLE IF NOT EXISTS pricing_config(
+   id                SERIAL PRIMARY KEY,
+   cfg   jsonb       NOT NULL,
+   created_at        timestamp NOT NULL default now()
+);
+CREATE TABLE IF NOT EXISTS proposals
+(
+   key               text PRIMARY KEY,
+   proposal          jsonb NOT NULL,
+   expires_at        timestamp NOT NULL,
+   COLUMN updated_at timestamp NOT NULL
+);
 ```
 
+#### DB Migrations
+
+Path for migration files: `db/migrations`
+
+All migrations are done on Client Init
+
+#### NATS Msg Broker channels
+
+* `*.proposal-register.v3` - Register new Node proposal
+* `*.proposal-ping.v3` - Node Ping
+* `*.proposal-unregister.v3` - Unregister Node proposal
+
+#### Entries
+
+* `cmd/main.go` - Discovery service
+* `sidecar/cmd/main.go` - Sidecar Pricing Parser service
+
+#### Code structure
+
+* `/config` - [Discovery] config parser. Env params
+* `/tags` - [Discovery] Tags proposals enhancer for SuperProxy 
+* `/docs` - [Discovery] Auto generated Swagger for REST API
+* `/e2e` - e2e tests
+* `/health` - [Discovery] health checker REST API 
+* `/listener` - [Discovery] NATS listener
+* `/price/api.go` - [Discovery] Pricing REST API
+* `/price/config.go` - [Discovery, Sidecar] Pricing config
+* `/price/market.go` - [Sidecar] Scheduled price fetcher from different APIs (Gecko, Coinmarket) 
+* `/price/network_load_calculator.go` - [Sidecar] Updating countries multipliers based on load (sessions/providers)
+* `/price/price_getter.go` - [Discovery] Fetch prices from Redis
+* `/price/price_updater.go` - [Sidecar] Scheduled price updater to Redis
+* `/proposal/metrics/echnancer.go` - [Discovery] Proposal quality enhancer
+* `/proposal/v3` - [Discovery] Proposal structs
+* `/proposal/api.go` - [Discovery] Proposal REST API
+* `/proposal/repository.go` - [Discovery] Mappings for Postgres
+* `/proposal/service.go` - [Discovery] Proposal service with scheduled expiration job
+* `/quality/oracleapi` - [Discovery] Quality Oracle REST API client
+* `/quality/service.go` -  [Discovery] Caching layer with BigCache for Quality Oracle responses
+
+## Development
+
+### What toolset needed for development
+
+https://docs.mysterium.network/developers/
+
+### How to build it
+
+`mage build` - Builds Discovery service
+
+`mage buildsidecar` - Builds Sidecar Prices Parser
+
+Or for docker 
+
+`docker build -t discovery:local .`
+
+`docker build -t discosidecar:local ./sidecar`
+
+### How to run it
+
+`mage up` - will run docker-compose with all needed services
+
+Or
+
+`docker-compose up`
+
+### How to test it locally
+
+`mage e2edev`
+
+### How to deploy it in production
+
+1. Push new version tag to the repo
+2. Change k8s manifest file for Discovery to use new tag and commit to git
+3. Click sync on ArgoCD dashboard for discovery 
```

Comment:
Isn't this a public project? Do we _really_ want to explain this here?

Reply:
It doesn't expose any secret information, so why not? @Snawoot Can you comment?

Reply:
I think what @tomasmik means is that for outside contributors knowledge about internal affairs won't make this more perceptive. Perhaps splitting this into `for outside contributors` and `internal team` on how to deploy would be better?

Reply:
Yes that's more or less what I meant. ^

Reply:
removed